ag_data/formulas/library/system/mercury_formulas.py

```diff
@@ -0,0 +1,50 @@
+#
+# Formulas for Mercury's built-in and natively supported sensor types
+# Sample processing formulas with primary sensors and fictional Mercury-branded sensors
+#
+
+from ag_data import models
+
+
+def fPass(timestamp, sensor, measurement_payload):
+    result = {}
+
+    return result
+
+
+def fMercurySimpleTemperatureSensor(timestamp, sensor, measurement_payload):
+    result = {}
+
+    return result
+
+
+def fMercuryDualTemperatureSensor(timestamp, sensor, measurement_payload):
+    mean = measurement_payload["internal"] / 2 + measurement_payload["external"] / 2
+    diff = measurement_payload["internal"] - measurement_payload["external"]
+
+    result = {"mean": mean, "diff": diff}
+
+    return result
+
+
+def fMercuryFlowSensor(timestamp, sensor, measurement_payload):
+    result = {}
+
+    latest = models.AGMeasurement.objects.filter(sensor_id=sensor.id)
```

Comment:
How about changing `latest -> measurement`. It's not actually latest until you call `latest.latest` at the line38 i guess. I think this will also make line 35 and 38 better.

Reply:
Will change the name. Does `measurements` sound good?

Reply:
Good!

Reply:
This is now in #376.